tests/vertexonly/test_interpolation_from_parent.py

```diff
@@ -132,17 +140,10 @@ def test_vector_function_interpolation(parentmesh, vertexcoords, vfs):
         if parentmesh.ufl_cell().cellname() == "quadrilateral":
             if not (vfs_fam == "RTCE" or vfs_fam == "RTCF"):
                 pytest.skip(f"{vfs_fam} does not support {parentmesh.ufl_cell()} cells")
-            # TODO: Remove this else when fixed
-            else:
-                pytest.skip(f"Some complex merge related problem for {vfs_fam}, get this from loopy: TypeError: unsupported type for persistent hash keying: <class 'complex'>")
```

Comment:
Is this actually fixed then? What fixed it?

Reply:
Oh sorry, I forgot I also had changes in finat: https://github.com/FInAT/FInAT/pull/92